includes/Core/Permissions/Permissions.php

```diff
@@ -163,31 +166,29 @@ public function __construct( Context $context, Authentication $authentication, M
 			self::MANAGE_OPTIONS      => 'manage_options',
 			self::SETUP               => 'manage_options',
 		);
-		// TODO Add the element assigned below into $this->base_to_core above when the dashboard sharing feature flag is removed.
-		if ( Feature_Flags::enabled( 'dashboardSharing' ) ) {
-			// Allow editors and up to view shared dashboard data.
-			$this->base_to_core[ self::VIEW_SHARED_DASHBOARD ] = 'edit_posts';
-		}
 
 		$this->meta_to_core = array(
 			// Allow users that can edit a post to view that post's insights.
 			self::VIEW_POST_INSIGHTS => 'edit_post',
 		);
 
 		$this->meta_to_base = array(
+			self::VIEW_SPLASH                  => self::VIEW_DASHBOARD,
+			self::VIEW_AUTHENTICATED_DASHBOARD => array( self::VIEW_DASHBOARD, self::AUTHENTICATE ),
 			// Allow users that can generally view posts insights to view a specific post's insights.
-			self::VIEW_POST_INSIGHTS => self::VIEW_POSTS_INSIGHTS,
+			self::VIEW_POST_INSIGHTS           => self::VIEW_POSTS_INSIGHTS,
 		);
 		// TODO Merge the array below into $this->meta_to_base above when the dashboard sharing feature flag is removed.
 		if ( Feature_Flags::enabled( 'dashboardSharing' ) ) {
 			$this->meta_to_base = array_merge(
 				$this->meta_to_base,
 				array(
-					// Allow users that can generally view the shared dashboard to read shared module data.
-					self::READ_SHARED_MODULE_DATA       => self::VIEW_SHARED_DASHBOARD,
+					// Allow users that can generally view dashboard to read shared module data.
+					self::READ_SHARED_MODULE_DATA       => self::VIEW_DASHBOARD,
 					// Admins who can manage options for SK can generally manage module sharing options.
 					self::MANAGE_MODULE_SHARING_OPTIONS => self::MANAGE_OPTIONS,
 					self::DELEGATE_MODULE_SHARING_MANAGEMENT => self::MANAGE_OPTIONS,
+					self::VIEW_SHARED_DASHBOARD         => self::VIEW_DASHBOARD,
```

Comment:
Just wondering: Don't these arrows need to align for coding standards?

Reply:
Weird. Anyway, not a big deal.

Reply:
Yes, but for some reason this is where PHPCS thinks they should be 🤷‍♂️